src/coreclr/jit/simdashwintrinsic.cpp

```diff
@@ -682,30 +641,11 @@ GenTree* Compiler::impSimdAsHWIntrinsicSpecial(NamedIntrinsic       intrinsic,
             break;
         }
 
-        case NI_VectorT128_Sum:
-        {
-            if (varTypeIsFloating(simdBaseType))
```

Comment:
I see that the case for `VectorT_Sum` is moved below. wondering we don't need `InstructionSet_*` checks anymore?

Reply:
The implementation was updated to not use the horizontal operations anymore, as it has better performance on modern hardware and was needed for Vector512_Sum anyways.